cypress/integration/about_page_spec.js

```diff
@@ -0,0 +1,128 @@
+/// <reference types="cypress" />
+
+describe("About page", () => {
+  beforeEach(() => {
+    cy.visit("/about");
+  });
+
+  it("displays main hero section with an image, title and subtitle", () => {
+    cy.get("[data-testid=hero-section]").within($hero => {
+      cy.get(".hero-image");
+      cy.get("[data-testid=hero-title]");
+      cy.get("[data-testid=hero-subtitle]");
+    });
+  });
+
+  it("displays correct breadcrumb navigation", () => {
+    cy.get("[data-testid=breadcrumbs]").within($breadcrumbs => {
+      cy.contains("Home");
+      cy.contains("About");
+    });
+  });
+
+  it("displays section: collection summary", () => {
+    cy.get("[data-testid=section-collection-summary]").within($section => {
+      cy.get("p");
+    });
+  });
+
+  it("displays section: collection highlights along with photo grid to display work", () => {
+    cy.get("[data-testid=collection-highlights-section]");
+    cy.contains("h3", "Collection Highlights");
+    cy.get("[data-testid=feature-two-col1]");
+    cy.get("[data-testid=feature-two-col2]");
+    cy.get("[data-testid=feature-two-col3]");
+    cy.get(
+      "[data-testid=feature-box-4ed2338d-c715-4a86-8ac6-6b4030a42be5]"
+    ).within($featurebox1 => {
+      cy.get("img");
+      cy.contains("h4", "Hamid Naficy Iranian Movie Posters Collection");
+      cy.get("p");
+      cy.get("a")
+        .contains("View Work")
+        .should("have.attr", "href");
```

Comment:
Let's add an additional `.and()` here to check that the `href` attribute contains part of the text/route that we expect.